===================================================================
RCS file: /web/pages/us.freeradius.org/cvs/radiusd/src/modules/rlm_smb/smblib.c,v
retrieving revision 1.3
retrieving revision 1.4
diff -u -p -r1.3 -r1.4
--- radiusd/src/modules/rlm_smb/smblib.c	2002/08/06 16:50:33	1.3
+++ radiusd/src/modules/rlm_smb/smblib.c	2003/11/25 16:03:17	1.4
@@ -316,7 +316,7 @@ int SMB_Logon_Server(SMB_Handle_Type Con
 
 { struct RFCNB_Pkt *pkt;
   int param_len, i, pkt_len, pass_len,a;
-  char *p, pword[128];
+  char *p, pword[256];
 
   /* First we need a packet etc ... but we need to know what protocol has  */
   /* been negotiated to figure out if we can do it and what SMB format to  */
